index.js

```diff
@@ -14,7 +14,12 @@ module.exports = function npmLinkCheck(pathToRoot, cb) {
 };
 
 function globNodeModules(startPath, cb) {
-    var pathToNodeModules = path.join(startPath, 'node_modules');
+    // if we have a scope we should check if the modules inside the folder is linked 
+    if (startPath.includes('@')) {
+        var pathToNodeModules = startPath;
+    } else {
+        var pathToNodeModules = path.join(startPath, 'node_modules');
```

Comment:
I'm not a fan of defining a `var` inside an if-else clause. 

Would you mind rewriting this as:

```js
var pathToNodeModules;

if (startPath.includes('@')) {
    pathToNodeModules = startPath;
} else {
    pathToNodeModules = path.join(startPath, 'node_modules');
}
```

or

```js
var pathToNodeModules = startPath.includes('@')
    ? startPath
    : path.join(startPath, 'node_modules');
```

?